pybossa/redis_lock.py

```diff
@@ -256,5 +260,6 @@ def acquire_reserve_task_lock(self, project_id, task_id, user_id, category):
 
 
     def release_reserve_task_lock(self, resource_id, pipeline, expiry):
-        cache = pipeline or self._redis
+        #cache = pipeline or self._redis # https://pythonrepo.com/repo/andymccurdy-redis-py-python-connecting-and-operating-databases#locks
+        cache = self._redis
```

Comment:
Are we not using pipeline here? Can we also remove it from the arguments?

Reply:
good point. thanks!